src/jsonValidator.js

```diff
@@ -0,0 +1,14 @@
+import { keyTypes } from './constants.js'
+import { createError } from './error.js';
+
+export const jsonValidator=(jsonString) => {
+  const err = [];
+
+  try {
+    const checkJson = JSON.parse(jsonString);
+  } catch (error) {
+    err.push(createError("invalid-json"));
+    return false;
+  }
+  return true;
+}
```

Comment:
Because of the return values of the function, i would say a more appropriate name for this would be something like `jsonIsValid`. 
As i see it, they way this would be used could be something like:
```
jsonIsValid(jsonString) ? doSomething : doSomethingElse
```

Otherwise, it's not really clear to me how this would be used.

Reply:
This sounds like a good improvement, I'll change it to make it clearer.